site/src/main/java/uk/nhs/digital/website/beans/EffectiveDate.java

```diff
@@ -26,4 +27,9 @@ public Calendar getEndDate() {
         return getProperty("website:enddate");
     }
 
+    @HippoEssentialsGenerated(internalName = "website:datescale")
+    public String getDatescale() {
```

Comment:
getDateScale() so consistent with rest of the file

Reply:
done